types/src/Experiment/enums.ts

```diff
@@ -18,7 +18,7 @@ export enum CONDITION_ORDER {
 
 export enum ASSIGNMENT_ALGORITHM {
   RANDOM = 'random',
-  STRATIFIED_RANDOM_SAMPLING = 'stratified random sampling',
+  STRATIFIED_RANDOM_SAMPLING = 'stratified_random_sampling',
```

Comment:
why do we need to change this?

Reply:
It was just to make the enums in consistent snake case format, for the same reason as I commented above.

Reply:
ok, as stated in the other comments, this change is kind of more than just a string change because it implicates the db schema, not necessary to do for this story.

Reply:
Resolved (commit: https://github.com/CarnegieLearningWeb/UpGrade/pull/2183/commits/c56efbc60fb5a5064763b38937e0467e4734b089)